profiles/common/modules/features/multisite_notifications/multisite_notifications_core/multisite_notifications_core.module

```diff
@@ -12,17 +12,11 @@ include_once 'multisite_notifications_core.features.inc';
  * Implements hook_subscriptions_queue_alter().
  */
 function multisite_notifications_core_subscriptions_queue_alter(&$event) {
-  // NB this only works in edit mode.
-  if (isset($event['node']->workbench_moderation['updating_live_revision'])
-    && $event['node']->workbench_moderation['updating_live_revision']
-  ) {
-    $event = NULL;
-  }
-
-  // Only send notification if the state of the content is published.
-  if (isset($event['node']->workbench_moderation_state_new)) {
-    // Content has a worbench moderation workflow.
-    if ($event['node']->workbench_moderation_state_new != workbench_moderation_state_published()) {
+  // Is workbench_moderation overriding the workflow?
+  if (isset($event['node']->workbench_moderation)) {
+    // Only process events after publishing
+    // See multisite_notifications_core_workbench_moderation_transition()
+    if (!isset($event['node']->flag_after_publishing)) {
```

Comment:
I've changed the name as suggested.
I don't see a practical advantage of setting the parameter to FALSE. But if you want to change that please do it, no problem also.